db/migrate/20151103223107_drop_properties.rb

```diff
@@ -0,0 +1,5 @@
+class DropProperties < ActiveRecord::Migration
+  def change
```

Comment:
This is a clean up step, right? I was following the previous precedents in db/migrate, which just `drop_table` and move on. We don't want to roll this back, ever, and couldn't if we wanted to, since the data is gone once the table is dropped. Restoring an empty table isn't really a reversible action.